BO4E/BO/Reklamation.cs

```diff
@@ -0,0 +1,102 @@
+﻿using BO4E.COM;
+using BO4E.EnergyIdentificationCodes;
+using BO4E.ENUM;
+using BO4E.meta;
+
+using Newtonsoft.Json;
+
+using ProtoBuf;
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace BO4E.BO
+{
+    /// <summary>
+    /// Reklamations BO
+    /// </summary>
+    [ProtoContract]
+    [NonOfficial(NonOfficialCategory.MISSING)]
+    public class Reklamation : BusinessObject
```

Comment:
wenn es sich um die reklamation von energiemengen handelt, wie in https://github.com/Hochfrequenz/go-bo4e/pull/88, dann sollten wir das glaube ich auch an der benamung deutlich machen (sieh euach kommentare am Go-PR)

Reply:
So wie ich es vorhin verstanden habe, wird es aktuell für die Energiemenge genutzt, soll aber generisch eine allgemeine Reklamation sein.